packages/@coorpacademy-player-web/package.json

```diff
@@ -36,6 +36,7 @@
     "@babel/plugin-proposal-object-rest-spread": "^7.4.3",
     "@babel/polyfill": "^7.4.3",
     "@babel/preset-env": "^7.4.3",
+    "@coorpacademy/css-modules-require-hook": "^2.0.1",
```

Comment:
?

Reply:
Needed because it was added 28 days ago and not be added in package.json (error in CI, look at [this](https://travis-ci.com/CoorpAcademy/components/jobs/197975092))

Reply:
What about all pRs that have been merged before this one, why didn't this test not fail?

Reply:
All other PR (modifying `player-web`) were failing: https://github.com/CoorpAcademy/components/commits/master

Note that Travis only runs tests on altered packages, so, if you didn't worked on `player-web`, it will stay green so.